connor/connor.go

```diff
@@ -40,6 +40,10 @@ func matchWith(op string, conditions, data any) (bool, error) {
 		return like(conditions, data)
 	case "_nlike":
 		return nlike(conditions, data)
+	case "_ilike":
+		return ilike(conditions, data)
+	case "_nilike":
```

Comment:
Moving this here to keep under a thread:
From @shahzadlone
> thought(out-of-scope): _nilike seems odd reading it, what if we prepend all negated labels with n so for example we would have _n_eq, _n_in, _n_ilike

Reply:
I do prefer `_nilike` over `_n_ilike` at first glance.

Reply:
Would it be possible to add a logical not operator to connor? Then we could remove the `_n` variants.

Reply:
It's just that the first three chars read `_nil`... haha

Reply:
> Would it be possible to add a logical not operator to connor? Then we could remove the _n variants.

Maybe like this?
```go
switch op {
	case "_and":
		return and(conditions, data)
	case "_eq":
		return eq(conditions, data)
	case "_ge":
		return ge(conditions, data)
	case "_gt":
		return gt(conditions, data)
	case "_in":
		return in(conditions, data)
	case "_le":
		return le(conditions, data)
	case "_lt":
		return lt(conditions, data)
	case "_ne": // -> "_!eq"
		return ne(conditions, data)
	case "_nin": // -> "_!in"
		return nin(conditions, data)
	case "_or":
		return or(conditions, data)
	case "_like":
		return like(conditions, data)
	case "_nlike": // -> "_!like"
		return nlike(conditions, data)
        case "_ilike":
		return ilike(conditions, data)
	case "_nilike": // -> "_!ilike"
		return nilike(conditions, data)
	case "_not":
		return not(conditions, data)
	default:
		return false, NewErrUnknownOperator(op)
	}
```

Reply:
Instead of `_nilike: "test"` it could be `_not: { _ilike: "test" }`

Reply:
> Instead of `_nilike: "test"` it could be `_not: { _ilike: "test" }`

This can already be done. `_nilike` is available for convenience.